tests/smoke_test.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+import ddtrace.bootstrap.sitecustomize as module
```

Comment:
this is probably enough for initial version, but I wonder what else we can/should do for testing?

like, maybe profiling Cython code compiles, but we should test loading it too?

Reply:
Definitely! Is there a module we can import to test cython code is correctly loading?

Reply:
This is a good question.... I am ok to defer for now, at least we have a way to do it.

Loading and using the msgpack encoder, generating a random id, or start the profiler could all be helpful ?